src/content/posts/web3py-solidity-write-up.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: "Write-up of The Art of Deception (Blockchain) CTF HTB Apocalypse 2023"
-summary: "Simple tutorial to Discord bots using `discord.py`"
+summary: "Introduction to Web3 security: an explanation of the logic put behind flagging a Web3 challenge, written in web3py and solidity."
 date: 2023-03-28T15:08:45-02:00
 lastUpdate: 2023-03-28T15:08:45-02:00
-tags: ["web3","solidity","write-up","Supwn"]
+tags: ["introduction", "web3", "solidity", "write-up", "Supwn"]
```

Comment:
maybe a CTF tag ?

Reply:
That's the role of the `write-up` tag in my opinion.
I was looking to reduce the number of tags, to only keep explicit, exhaustive and useful ones because I feared that using redundant tags would just harm and waste the interest of the tag system in the first place.